modules/nw-rfhe-subscribing-redfish-events.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-troubleshooting_{context}"]
+= Subscribing to Redfish Events
+
+To subscribe to Redfish Events, create a BMCEventSubscription CRD.
+
+[source,yaml]
+----
+    apiVersion: metal3.io/v1alpha1
+    kind: BMCEventSubscription
+    metadata:
+      name: worker-1-events
+    spec:
+       hostName: ostest-worker-1 <1>
+       destination: https://events.apps.corp.example.com/webhook <2>
+       context: “SomeUserContext”
+       httpHeadersRef:
+         name: some-secret-name
+         namespace: some-namespace
+    status:
+      errorMessage: ""
+      subscriptionID: aa618a32-9335-42bc-a04b-20ddeed13ade
+----
```

Comment:
The CR to be created should look like this. The above text with `status and subscriptionID` is the result of applying this CR.
```
apiVersion: metal3.io/v1alpha1
kind: BMCEventSubscription
metadata:
  name: sub-01
  namespace: some-namespace
spec:
   hostName: baremetal-host-name
   destination: https://events.apps.corp.example.com/webhook
   context: “SomeUserContext”
```

Reply:
Topic now in scope of https://github.com/openshift/openshift-docs/pull/41832.